.github/workflows/ci-jdk11.yml

```
@@ -22,8 +22,24 @@ name: Java CI with Maven On JDK 11
 on:
   push:
     branches: [ "master" ]
+    paths:
+      - '**.java'
+      - '**.xml'
+      - '**.yml'
+      - '**.yml'
+      - '**.sql'
+      - '**.properties'
   pull_request:
     branches: [ "master" ]
+    paths-ignore:
+      - '**.java'
+      - '**.xml'
+      - '**.yml'
+      - '**.yml'
```

```suggestion
      - '**.yml'
```

The same issues as above.